tutorial/markdown/nodejs/express.md

```diff
@@ -22,343 +21,335 @@ sdk_language:
 length: 30 Mins
 ---
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=nodejs-quickstart-repo&source=devPortal)](https://gitpod.io/#https://github.com/couchbase-examples/nodejs-quickstart)
+In this tutorial, you will learn how to connect to a Couchbase Capella cluster to create, read, update, and delete documents and how to write simple parametrized SQL++ queries.
 
-In this article, you will learn how to connect to a Couchbase cluster to create, read, update, and delete documents, and write simple parametrized N1QL queries using the Couchbase Node JS SDK.
-
-We will be using the latest version of Couchbase (version 7) that enables scopes and collections. For the easiest setup experience, we recommend trying Couchbase Capella, our fully-managed DBaaS offering. [Claim your free trial!](https://cloud.couchbase.com/sign-up)
-
-Alternatively, you can [install Couchbase with docker](https://docs.couchbase.com/server/current/getting-started/do-a-quick-install.html) or [directly on your local device](https://docs.couchbase.com/server/current/install/install-intro.html).
 
 ## Prerequisites
 
 To run this prebuilt project, you will need:
 
-- A Couchbase Capella cluster or Couchbase 7 installed locally
-- NodeJS & NPM (v12+)
-- Code Editor
-- See the [Couchbase Installation Options](/tutorial-couchbase-installation-options) for more details on setting up a cluster
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/nodejs-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- NodeJS & NPM 
```

Comment:
Can you also add the Minimum version of Node supported?

Reply:
Added Node js version LTS as discussed